### PR TITLE
Update openapi jars [ENT-5076]

### PR DESCRIPTION
**Pull request by wottop**
Used 2 commits so that the update from the new openapi version would be separate from our additions.

### CODE REVIEW

**Review by nikosmoum — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/java/org/openapitools/codegen/languages/AbstractJavaJAXRSServerCodegen.java [hunk 1]
```diff
@@ -1,318 +0,0 @@
...
-import org.openapitools.codegen.languages.features.BeanValidationFeatures;
-import org.openapitools.codegen.utils.URLPathUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.File;
-import java.net.URL;
-import java.util.*;
-
-public abstract class AbstractJavaJAXRSServerCodegen extends AbstractJavaCodegen implements BeanValidationFeatures {
```
[the remark this answers is not in the document]
Comment (nikosmoum) [reply buildSrc/src/main/java/org/openapitools/codegen/languages/AbstractJavaJAXRSServerCodegen.java:35]: And now that I think about it, we also need to do the same for each of the .mustache template files we're overriding (api.mustache, formParams.mustache, pojo.mustache, returnTypeInterface.mustache). We're basically rebasing our custom changes on them to the 6.0.0 versions from here: https://github.com/OpenAPITools/openapi-generator/tree/6.0.x/modules/openapi-generator/src/main/resources/JavaJaxRS/spec

**Comment by nikosmoum** [comment]
Unsure if it's the exact same issue, but it may be this one: https://github.com/OpenAPITools/openapi-generator/issues/12549

**Comment by nikosmoum** [comment]
Also, can we get the commit message to include the upgraded version (6.0.0)?

**Comment by wottop** [comment]
The pojo.mustache file was added to the client because of bugs in the implementation of 6.0.0

**Comment by nikosmoum** [comment]
As a note for the future: Whenever the fix for https://github.com/OpenAPITools/openapi-generator/issues/12549 is available in a version we can upgrade to, we should remove the pojo.mustache template workaround.

**Review by nikosmoum — CHANGES_REQUESTED** [review]
Can we squash this and also make the commit message more specific e.g. 'Upgrade to openapi-generator 6.0.0' which helps searching the git log

**Review by nikosmoum — APPROVED** [review]

**Pull request merged**